portality/static/js/application_form.js

```diff
@@ -844,8 +844,8 @@ window.Parsley.addValidator("optionalIf", {
 });
 
 window.Parsley.addValidator("differentTo", {
-    validateString : function(value, requirement) {
-      return (!value || ($("[name = " + requirement + "]")).val() !== value);
+    validateString : function(value, requirement, message) {
+      return (!value || ($("[name = " + requirement + "]")).val().toLowerCase() !== value.toLowerCase());
```

Comment:
This function takes the argument `message` but doesn't use it, is that required by the Parsley interface or something?

Reply:
No, that's overlook. Sorry!